source/reference/minio-server/minio-server.rst

```diff
@@ -171,6 +171,54 @@ The command accepts the following arguments:
    Omit to direct MinIO to generate a dynamic port at server startup. The
    MinIO server outputs the port to the system log.
 
+.. mc-cmd:: --ftp
+   :optional:
+   
+   Enable and configure an File Transfer Protocol (``FTP``) or File Transfer Protocol over SSL/TLS (``FTPS``) server.
+   Use multiple times to specify an address port or a passive port range of addresses as key-value pairs.
+
+   Valid keys:
+
+   - ``address``, which takes a single port to use for the server, typically ``8021``
+   - ``passive-port-range``, which restricts the range of potential ports the system assigns to FTP
+   - ``tls-private-key``, which takes the path to the user's private key for accessing the MinIO deployment by TLS
+   - ``tls-public-cert``, which takes the path to the certificate for accessing the MinIO deployment by TLS
```

Comment:
These two are optional, I assume.

Reply:
Yes, updated to indicate that. And the passive port range.